.github/workflows/build.yml

```diff
@@ -42,6 +42,23 @@ jobs:
         echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
         docker push antrea/antrea-ubuntu:latest
 
+  build-scale:
+    needs: check-changes
+    if: ${{ needs.check-changes.outputs.has_changes == 'yes' || github.event_name == 'push' }}
+    runs-on: [ubuntu-18.04]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build Antrea Docker image
+      run: make build-scale-simulator
+    - name: Push Antrea Docker image to registry
```

Comment:
step names are wrong, these were copy-pasted from the job above

Reply:
done